tests/classes/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -75,16 +74,18 @@ class Readable extends events.EventEmitter implements NodeJS_ReadableStream {}
 class InMemoryAppender extends log4javascript.Appender
   implements ICachedLogMessageProvider {}
 
-class Foo extends Immutable.Record({
-  ipaddress: ""
-}) {
+class Foo
+  extends Immutable.Record({
+    ipaddress: ""
+  }) {
```

Comment:
Fwiw, we had the behavior you are changing it to in the early days of prettier but I changed to what we have now for this example and extending React component with types. It looks better in one line.

Now, this causes a bunch of problematic things (like all the rest that’s being fixed by this pr) so it may not be a bad idea to follow through. I just wanted to give some context

Reply:
Could we inline only if it is a `CallExpression`?

Reply:
What could be a good way is to keep the previous implementation if there’s just a single extend. Otherwise use your new logic

Reply:
React.Component<...> is not a call expression

Reply:
We do want a long identifier to break, though.

Reply:
Once you decided when not to break (which expression), i will continue with this :)

Reply:
@azz @vjeux @j-f1 
Any update on this?
I mean if its a long statement it should def brake, and it makes it def more readable like this.
If we limit it to only break when there is a single extends
something like this 
class AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA extends AAAAAAAAAAAAAA
would still look shitty.